MC/config/analysis_testing/json/analysis-testing-data-qa-event-track.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
could wen call this file maybe
`analysis-testing-EventTrackQA-data.json`
to make the connection between the analysis task and JSON as clear as possible?
(same in case of MC)

Reply:
done